packages/util-address/types/index.d.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * @description Removes 0x from address if present
+ * @param {string} address - Flow address
+ * @returns {string} - Flow address without 0x prefix
+ */
+export function sansPrefix(address: string): string;
```

Comment:
Given this is a build product akin to the dist folder can we ignore these files in .gitignore?

Reply:
It would make the build step easier, but don't most libraries have their type files committed?

Reply:
Not if they are generated. Usually, they go into the dist folder.

Reply:
If you do find an example of what we are doing here out in the wild, I would be curious to see it.